SPECS/qemu-dp.spec

```diff
@@ -137,6 +143,9 @@ rm -rf %{buildroot}/usr/include %{buildroot}%{_libdir}/pkgconfig %{buildroot}%{_
 %{_libdir}/qemu-dp/bin
 
 %changelog
+* Thu Jul  5 2018 rposudnevskiy <ramzes_r@yahoo.com> - 2.10.2-1.2.1
```

Comment:
According to http://www.tldp.org/HOWTO/RPM-HOWTO/build.html the date must be as output by `LC_ALL=C date +"%a %b %d %Y"`. This means that we need the leading 0 to the day number.